venv/CalculationClass.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
```

Comment:
Use 'snake_case` for file names. Do note that mistakes will be pointed out only once. You will be assessed on how well you can incorporate suggestions into your projects.